edxval/tests/test_api.py

```diff
@@ -2885,6 +2885,39 @@ def test_create_transcript_file(self):
         transcript = api.get_video_transcript_data(video_id=video_id, language_code=language_code)
         self.assertEqual(transcript['content'], expected_transcript_content)
 
+    def test_unpublished_create_transcript_file(self):
+        """
+        Tests that an unpublished transcript file is created correctly.
+        """
+        language_code = 'en'
+        video_id = constants.VIDEO_DICT_FISH['edx_video_id']
+        transcript_file_name = u'super-soaker-en.srt'
+        expected_transcript_path = combine(
+            combine(self.temp_dir, constants.EXPORT_IMPORT_COURSE_DIR),
+            combine(constants.EXPORT_IMPORT_STATIC_DIR, transcript_file_name)
+        )
+
+        delegate_fs = OSFS(self.temp_dir)
+        file_system = delegate_fs.makedir(constants.EXPORT_IMPORT_COURSE_DIR, recreate=True)
+        # static_dir isn't made when the transcript is not published
```

Comment:
Instead of inline comments, explain the scenario in unit test docstring

Reply:
I'll move it into the docstring.